.github/workflows/pre-submit.reviewdog.yml

```diff
@@ -1,5 +1,7 @@
 name: reviewdog
-on: [pull_request]
+
+"on": [pull_request_target]
```

Comment:
if you don't define the permissions, they are `write-all` by default. Define the exact permission you need access to, something like:
```
permissions:
  // others will default to `none`
  statuses: write
```

Reply:
Yeah, I guess I can just set this to empty since I'm using a PA? The linter actions seem to use the [checks API](https://docs.github.com/en/rest/checks) but for whatever reason there aren't any scopes you can add to Github Actions to allow it so you need to use a PA.

As an aside, we should probably also consider setting the default permissions to the "restrictive" set here. I think this is the "Read repository contents permission" setting under "Actions > general"
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Done.